src/test/resources/clustered-sasl-cs.xml

```diff
@@ -0,0 +1,260 @@
+                        <property name="use_mcast_xmit">false</property>
+                    </protocol>
+                    <protocol type="UNICAST3"/>
+                    <protocol type="pbcast.STABLE"/>
+                    <protocol type="pbcast.GMS"/>
+                    <protocol type="MFC"/>
+                    <protocol type="FRAG3"/>
+                </stack>
+                <stack name="tcp-gossip">
+                    <transport type="TCP" socket-binding="jgroups-tcp"/>
+                    <protocol type="TCPGOSSIP">
+                        <property name="initial_hosts">${jgroups.gossip.initial_hosts:}</property>
+                    </protocol>
+                    <protocol type="MERGE3"/>
+                    <protocol type="FD_SOCK" socket-binding="jgroups-tcp-fd"/>
+                    <protocol type="FD_ALL"/>
+                    <protocol type="VERIFY_SUSPECT"/>
+                    <protocol type="pbcast.NAKACK2">
+                        <property name="use_mcast_xmit">false</property>
+                    </protocol>
+                    <protocol type="UNICAST3"/>
+                    <protocol type="pbcast.STABLE"/>
+                    <protocol type="pbcast.GMS"/>
+                    <protocol type="MFC"/>
+                    <protocol type="FRAG3"/>
+                </stack>
+            </stacks>
+        </subsystem>
+        <subsystem xmlns="urn:infinispan:server:core:9.0" default-cache-container="clustered">
+            <cache-container name="clustered" default-cache="default" statistics="true">
+                <transport lock-timeout="60000"/>
+                <security>
+                    <authorization>
+                        <identity-role-mapper/>
+                        <role name="admin" permissions="ALL"/>
+                        <role name="reader" permissions="READ"/>
+                        <role name="writer" permissions="WRITE"/>
+                        <role name="supervisor" permissions="READ WRITE EXEC BULK_READ BULK_WRITE LISTEN"/>
+                    </authorization>
+                </security>
+                <global-state/>
+                <distributed-cache name="default"/>
+                <distributed-cache name="authCache">
+                    <security>
+                        <authorization enabled="true" roles="admin reader writer supervisor"/>
+                    </security>
+                </distributed-cache>
+            </cache-container>
+        </subsystem>
+        <subsystem xmlns="urn:infinispan:server:endpoint:9.0">
+            <hotrod-connector socket-binding="hotrod" cache-container="clustered">
+                <topology-state-transfer lazy-retrieval="false" lock-timeout="1000" replication-timeout="5000"/>
+                <authentication security-realm="ApplicationRealm">
+                    <sasl mechanisms="DIGEST-MD5 PLAIN" qop="auth" server-name="${server.name:node0}" strength="high medium low">
```

Comment:
sasl server-name attribute must be the same for all the cluster nodes.
server-name="node0" should be fine